packages/airnode-deployer/config/config.json.example

```diff
@@ -29,8 +28,11 @@
     },
     "logFormat": "plain",
     "logLevel": "INFO",
-    "nodeVersion": "0.2.2",
-    "region": "us-east-1",
+    "nodeVersion": "0.3.0",
+    "cloudProvider": {
+      "name": "aws",
```

Comment:
if possible values for "name" are `local`, `aws`, `gcp`, etc, shouldn't we use "type" instead of "name"? I only ask because my concern is that someone might enter things like `aws_prod` or `my_pretty_aws_infra` as values since the key is called "name".

Reply:
That's a good idea, I'll rename it.